portalwire/portal_protocol.go

```diff
@@ -617,17 +621,19 @@ func (p *PortalProtocol) processOffer(target *enode.Node, resp []byte, request *
 	} else {
 		resultChan = nil
 	}
-	contentKeyBitlist := bitfield.Bitlist(accept.ContentKeys)
-	if contentKeyBitlist.Len() != uint64(contentKeyLen) {
+	acceptIndices := accept.GetAcceptIndices()
+	if accept.GetKeyLength() != contentKeyLen {
```

Comment:
The original `contentKeyLen` variable name is not good, it is easy to misunderstand it as a length of a single key. But it actually represents number of contentKeys, may be `contentKeysNum` better.